<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QMainWindow" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Futura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Home Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(210, 180, 140);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tutorialButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 136, 132);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="levelOneButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 136, 132);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="levelTwoButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 136, 132);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="levelThreeButton">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>201</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 136, 132);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>154</green>
          <blue>91</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>229</red>
          <green>228</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>38</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 154, 91);
color: rgb(229, 228, 226);
border: 4px solid rgb(128, 128, 0);
border-top-left-radius :20px;
 border-top-right-radius : 20px; 
 border-bottom-left-radius : 20px; 
 border-bottom-right-radius : 20px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Plant Doctor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tutorialLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 154, 91);
color: rgb(229, 228, 226);
border: 2px solid rgb(128, 128, 0);
border-top-left-radius :20px;
 border-top-right-radius : 20px; 
 border-bottom-left-radius : 20px; 
 border-bottom-right-radius : 20px;</string>
    </property>
    <property name="text">
     <string>Tutorial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelOneLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 154, 91);
color: rgb(229, 228, 226);
border: 2px solid rgb(128, 128, 0);
border-top-left-radius :20px;
 border-top-right-radius : 20px; 
 border-bottom-left-radius : 20px; 
 border-bottom-right-radius : 20px;</string>
    </property>
    <property name="text">
     <string>Level One</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelTwoLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 154, 91);
color: rgb(229, 228, 226);
border: 2px solid rgb(128, 128, 0);
border-top-left-radius :20px;
 border-top-right-radius : 20px; 
 border-bottom-left-radius : 20px; 
 border-bottom-right-radius : 20px;</string>
    </property>
    <property name="text">
     <string>Level Two</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelThreeLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 154, 91);
color: rgb(229, 228, 226);
border: 2px solid rgb(128, 128, 0);
border-top-left-radius :20px;
 border-top-right-radius : 20px; 
 border-bottom-left-radius : 20px; 
 border-bottom-right-radius : 20px;</string>
    </property>
    <property name="text">
     <string>Level Three</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
